config/config_params.go

```diff
@@ -213,6 +213,11 @@ type Config struct {
 	IpInIpMtu        int    `config:"int;1440;non-zero"`
 	IpInIpTunnelAddr net.IP `config:"ipv4;"`
 
+	// Knobs provided to explicitly control whether we add rules to drop encap traffic
+	// from workloads. We always add them unless explicitly disabled.
+	DropVXLANPacketsFromWorkloads bool `config:"bool;true"`
```

Comment:
Are most of our other fields default-zero-value?

Would `AllowVXLANPacketsFromWorkloads` be a better formulation of this? I think it would make it easier for golang users of the API (default value matches the type zero-value)

Reply:
Updated.